.zenodo.json

```diff
@@ -0,0 +1,44 @@
+{
+    "title": "SSCxEModelExamples",
+    "license": "CC-BY-4.0 license",
+    "upload_type": "software",
+    "description": "Repository for reproducing the work described at the 'A universal workflow for creation, validation and generalization of detailed neuronal models' publication.",
+    "creators": [
+        {
+            "affiliation": "Blue Brain Project, EPFL",
```

Comment:
Please also add the orcid ids

Reply:
How can I get them?

Reply:
Some of them are mentioned in other zenodo files of our packages. And for others you have to search for the person here: https://orcid.org/

Reply:
Alright, all 9 are added